charts/intel-gpu-resource-driver/values.yaml

```diff
@@ -37,6 +35,13 @@ kubeletPlugin:
       effect: "NoSchedule"
   affinity: {}
 
+cdi:
+  staticPath: /etc/cdi
+  dynamicPath: /var/run/cdi
+
+nodeFeatureRules:
+  enabled: false
+
 nfd:
   enabled: false # change to true to install NFD to the cluster
```

Comment:
NFD settings should be under same heading[1], e.g:
```
nfdRules:
  use: true
  install: true
```

Then you can use just:
`{{- if .Values.nfdRules.use }}`
instead of:
`{{- if or .Values.nodeFeatureRules.enabled .Values.nfd.enabled }}`

----

[1] If Helm variable names are changed, they should be changed consistently for all 3 Intel DRA helm charts (gaudi, gpu, qat), not just GPU one.

Reply:
You pass all values under the nfd key through to the nfd helm chart, so I did not want to potentially break something by passing nonexisting values through to nfd. I added a new key for backwards compatibility with the previous behavior. Let me know if this is fine or if you want a breaking change.

If you want to keep all three helm charts consistent, have you considered making it one chart?

Reply:
Good point.  Having 2 separate variable hierarchies for NFD is rather ugly though => @oxxenix do you see problems in adding/changing variables under `nfd`?

> If you want to keep all three helm charts consistent, have you considered making it one chart?

Not really my decision as I'm not involved with this project currently, except for doing a bit of review for the PRs.  @byako, any comments on this?

Reply:
I'm fine with this proposal.